### PR TITLE
Superseded bunk Crocker Highland neighbourhood record

### PR DESCRIPTION
**Pull request by stepps00**
Fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/771.

This PR includes:

- Adds an `edtf:cessation` date, `mz:is_current`, and `wof:superseded_by` value to the Piedmont record
- Adds a `wof:supersedes` value to the Oakland record
- Updates the Oakland record's `lbl:latitude` and `lbl:longitude` values to reflect the values in the Piedmont record
- Brought over any missing `misc`, `wk`, `src`, `mz`, and `wof` properties from the Piedmont record to the Oakland record

### CODE REVIEW

**Comment by stepps00** [comment]
The only descendants that are affected by this PR will be `venue` records in Piedmont ([list here](https://whosonfirst.mapzen.com/spelunker/download/85872275/?exclude=nullisland#14/37.8216/-122.2284)). We have not been updating `venue` hierarchies during any PIP work, so I think we're safe merging this PR as-is, once approved.

**Comment by thisisaaronland** [comment]
We have been updating venues, I just do it as a separate process.

File: data/858/722/75/85872275.geojson [hunk 1]
```diff
@@ -2,8 +2,9 @@
   "id": 85872275,
   "type": "Feature",
   "properties": {
-    "edtf:cessation":"uuuu",
+    "edtf:cessation":"2017-05-02",
```
Comment (nvkelso) [inline data/858/722/75/85872275.geojson]: Should the superseded record get **both** a `edtf:superseded` date and `edtf:cessation`?

Otherwise LGTM.

Comment (stepps00) [inline data/858/722/75/85872275.geojson]: The `edtf:superseded` property is just letting us know the date of superseding.. This is consistent with other superseding work we've done.. see: https://github.com/whosonfirst-data/whosonfirst-data/pull/752/files

**Comment by stepps00** [comment]
PR is ready to merge.

File: data/858/722/75/85872275.geojson (hunk 1 above)
Comment (nvkelso) [inline data/858/722/75/85872275.geojson]: So `superseded` is supplementary to `cessation`.

**Comment by nvkelso** [comment]
Over to @thisisaaronland.

**Pull request merged**